### ISSUE TITLE
Testing & Code Reviews

### ISSUE THREAD

**Issue by hbroer** [owja/ioc#8]
I could need some help with testing and some people doing code review of my own work. Feel free to open Issue, make pull requests or just comment here. :)

**Comment by levibostian** [comment]
I'm subscribed on GitHub to this project. I enjoy keeping up with the progress. Interested in me reading over pull requests and issues for you?

**Comment by hbroer** [comment]
Yea. I love to discuss code and ideas. :) First I need to add some inline documentation I think: #33. Because I have not that much time still it may take a while, but we will see. I really appreciate your interest in this little project.

**Comment by levibostian** [comment]
Absolutely. I love this little project because, well, it is little 😃. I found it while trying to find a DI library that was small in size for my projects. Inversify was cool, but too bloated for my needs. The vision outlined in the readme for this project immediately got me to love it. I love the vision and will do my best to stand by it.
